src/AspNetCoreModuleV2/CommonLib/HandleWrapper.h

```diff
@@ -51,6 +51,13 @@ class HandleWrapper
 
     HandleType* operator&() { return &m_handle; }
 
+    HandleType release() noexcept
+    {
+        auto value = m_handle;
```

Comment:
Reason for this change?

Reply:
Because we use it now to transfer handle ownership from LoadRequestHandlerAssembly to ApplicationFactory.